tasks/eks.py

```diff
@@ -70,9 +81,9 @@ def create_eks(
 
 def _show_connection_message(ctx: Context, full_stack_name: str, config_path: Optional[str]):
     outputs = tool.get_stack_json_outputs(ctx, full_stack_name)
-    kubeconfig_output = outputs["kubeconfig"]
+    kubeconfig_output = json.loads(outputs["dd-Cluster-aws-eks"]["kubeConfig"])
```

Comment:
This is based on the new Pulumi status output introduced in https://github.com/DataDog/test-infra-definitions/pull/484

```
 Outputs:
  - agent-linux-helm-install-name  : "dda"
  - agent-linux-helm-install-status: {
      - appVersion: "7"
      - chart     : "datadog"
      - name      : "dda"
      - namespace : "datadog"
      - revision  : 1
      - status    : "deployed"
    }
  - dd-Cluster-aws-eks             : {
      - kubeConfig : (json) {
          - apiVersion     : "v1"
          - clusters       : [
          -     [0]: {
                  - cluster: {
```